client/my-sites/plans/plan-feature/style.scss

```diff
@@ -15,64 +15,64 @@
 	font-weight: 300;
 }
 
-.feature-analytics__logo {
+.plan-feature__logo {
 	margin: 48px auto 48px;
 	text-align: center;
 }
 
-.feature-analytics__ilustration {
+.plan-feature__ilustration {
 	margin: 0 -24px -16px;
 
 	svg {
 		vertical-align: bottom;
 	}
 }
 
-.feature-analytics__footer {
+.feature-footer__wrapper{
 	border-top: 1px solid lighten( $gray, 20 );
 	border-bottom: 1px solid lighten( $gray, 20 );
 	margin: 48px 0;
 	padding: 16px;
 	display: flex;
 }
 
-.feature-analytics__footer-title {
+.feature-footer__title {
 	font-size: 15px;
 	font-weight: 600;
 	color: darken( $gray, 20% );
 	display: block;
 }
 
-.feature-analytics__refunds,
-.feature-analytics__support {
+.feature-footer__refunds,
+.feature-footer__support {
 	width: 50%;
 	text-align: center;
 	padding: 8px 24px;
 }
 
-.feature-analytics__footer-info {
+.feature-footer__info {
 	font-size: 12px;
 	color: $gray;
 	display: block;
 	margin-top: 8px;
 }
 
-.feature-analytics__support:hover {
-	.feature-analytics__footer-info,
-	.feature-analytics__footer-title {
+.feature-footer__support:hover {
+	.feature-footer__info,
+	.feature-footer__title {
 		color: $blue-medium;
 	}
 }
 
-.feature-analytics__plan-compare {
+.plan-feature__plan-compare {
```

Comment:
We can probably create a PlanComparisonGroup component, and have this css live with that. @mtias, were you thinking of something along those lines?